cmd/dockerd/daemon.go

```diff
@@ -306,7 +307,8 @@ func (cli *daemonCLI) start(ctx context.Context) (err error) {
 	}
 
 	routers := buildRouters(routerOpts)
-	httpServer.Handler = apiServer.CreateMux(routers...)
+	routers = append(routers, debugrouter.NewRouter())
```

Comment:
Actually; let me just move this inside `buildRouters`, as the order in which these are added shouldn't matter.